bin/execution.ml

```diff
@@ -63,6 +87,38 @@ let rec evaluate_one_step mode expression_list stack_ =
             let sum = v.value / u.value in
             let i = Integer { value = sum } in
             stack := Stack.push i stack''
+        | Float v1, Float v2 ->
+            let sum = v1.value /. v2.value in
+            let result = Float { value = sum } in
+            stack := Stack.push result stack''
+        | Integer v, Float u ->
+            let sum = float_of_int v.value /. u.value in
+            let result = Float { value = sum } in
+            stack := Stack.push result stack''
+        | Float v, Integer u ->
+            let sum = v.value /. float_of_int u.value in
+            let result = Float { value = sum } in
+            stack := Stack.push result stack''
+        (*do the rest*)
+        | _, _ -> failwith "not implemented")
+    | '%' -> (
+        match (operand1, operand2) with
+        | Integer v, Integer u ->
+            let sum = v.value mod u.value in
+            let i = Integer { value = sum } in
+            stack := Stack.push i stack''
+        | Float v1, Float v2 ->
```

Comment:
why do you use v1 and v2 here and not v and u (it does not really matter, just looks strange (also in the above parts))